src/deserialize/parse.rs

```diff
@@ -54,6 +52,23 @@ fn read_str(chars: &mut std::str::Chars) -> Edn {
     Edn::Str(string)
 }
 
+fn read_symbol(a: char, chars: &mut std::str::Chars) -> Result<Edn, Error> {
+    let c_len = chars
+        .clone()
+        .enumerate()
+        .take_while(|&(i, c)| i <= 200 && !c.is_whitespace() && c != ')' && c != '}' && c != ']')
+        .count();
+    match a {
+        ws if ws.is_whitespace() => Err(Error::ParseEdn(format!("\"{}\" could not be parsed", ws))),
+        c => {
+            let mut symbol = String::from(c);
+            let symbol_chars = chars.take(c_len).collect::<String>();
+            symbol.push_str(&symbol_chars);
+            Ok(Edn::Symbol(symbol))
+        }
+    }
```

Comment:
(style suggestion)
Do you think we will have more cases?
What do you think of 

```rust
if a.is_whitespace() {
    return Err(Error::ParseEdn(format!("\"{}\" could not be parsed", ws)));
}
let mut symbol = String::from(c);
let symbol_chars = chars.take(c_len).collect::<String>();
symbol.push_str(&symbol_chars);
Ok(Edn::Symbol(symbol))
```

Reply:
Actually `c` and `c_len` variables don't exist on my example.

Reply:
this was my first idea, but then we changed to `read_symbol` and it makes no more sense to have a match...

Reply:
92520e8